packages/terra-site/src/Index.jsx

```diff
@@ -121,6 +122,7 @@ ReactDOM.render((
       <Route path="time-input" component={TimeInputExamples} />
       <Route path="toggle" component={ToggleExamples} />
       <Route path="toggle-button" component={ToggleButtonExamples} />
+      {FormIndexRoutes}
     </Route>
```

Comment:
At some point in the future it might be nice to have the side navigation scroll independently so you don't have to scroll back up after making a selection.